test_plans/khr/free_function_commands.asciidoc

```diff
@@ -0,0 +1,413 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for sycl_khr_free_function_commands
+
+This is a test plan for the API described in
+https://github.com/KhronosGroup/SYCL-Docs/pull/644[sycl_khr_free_function_commands].
+
+
+== Testing scope
+
+=== Device coverage
+
+All of the tests described below are performed only on the default device that
+is selected on the CTS command line.
+
+=== Feature test macro
+
+All of the tests should use `#ifdef SYCL_EXT_KHR_FREE_FUNCTION_COMMANDS` so they can be skipped
```

Comment:
We drop the `_EXT_` from KHR extensions, and just use `_KHR_`.

```suggestion
All of the tests should use `#ifdef SYCL_KHR_FREE_FUNCTION_COMMANDS` so they can be skipped
```

Reply:
Thanks, copy-paste heritage. Updated.